src/components/select/Demo.vue

```diff
@@ -38,6 +43,7 @@
                   label="Multi select"
                   :error="error"
                   :large="large"
+                  :loading="loading"
```

Comment:
@sixteenstudio please see point 3

Reply:
> I don't think it I have ever implemented that because it isn't a thing on the html select. (this package has little to do with vue-ui)

Very possible I'm remembering the vue-ui select. Out of scope of this PR, but we should consider adding this as a future feature. It's incredibly helpful for the end-user and suffers no downsides.

> Okay so if it's not clearable then the select none should not be there if there's only one item selected on multi-select (Meaning this would be a bug). The cross icon only coming up when multiple is selected, is very much expected. Because we don't want to allow deselecting if there isn't anything else selected and it isn't clearable

Seems like we're overlapping `clearable` with `required`. The intention of `clearable` is to toggle a UI element that has the potential to cause issues with the design, that is, having an "x" hard coded on every input is very undesirable. It's a shortcut to selecting the value and removing it manually, that's all. I can, after all, delete all the content of a text box even if clearable is disabled.

The "Select None" is not a potential UI issue - it's hidden away inside a single component, so would argue it's safe to show at all times. While it's effect is the same, it has no relation to `clearable`.

> @sixteenstudio please see point 3

I think you are safe to make a decision on this yourself, been a while since this thread was started.

Reply:
None of the above needs to be actioned in this PR. Best if we extract it to a dedicated ticket.

Reply:
1.: https://github.com/karnama/vueish/issues/468
2.: see the message on discord
3.: this would only be a case for multiselect, and this isn't the behaviour for the html multiselects. I think lets keep as close as possible to the built in behaviour so the user already knows what to expect.